conda-build/Dockerfile

```diff
@@ -35,6 +34,7 @@ RUN bash ./install_conda.sh && rm install_conda.sh
 
 RUN anaconda login --username $AIHABITAT_CONDA_CHN --password $AIHABITAT_CONDA_CHN_PWD
 RUN conda create --name py36 python=3.6 -y && conda init bash
+RUN conda create --name py37 python=3.7 -y
```

Comment:
I don't believe this is needed.  `condo-build` takes are of setting up a py3.7 environment when you build a py3.7 package.

Reply:
@erikwijmans, good catch. Noticed that as well.

Reply:
Can we also remove the python=3.6 as well?

Reply:
No, we can't. That's root environment that we use to run conda build that has all dependency installed.